lib/tokenize.cpp

```diff
@@ -4508,6 +4508,15 @@ void Tokenizer::setVarId()
     for (Token *tok = list.front(); tok; tok = tok->next()) {
         if (tok->isName())
             tok->varId(0);
+        // commas in init..
+        if (!Token::Match(tok, "%name%|= {"))
```

Comment:
We used to check for `{` with `astOperand1()`, but the AST is not available yet.
This should probably handle `int a[]{1,2,3};` also, but we don't do that currently either.

Reply:
This will also match `class C {`, which should be avoided.